src/stories/SegmentedPicker/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import clsx from "clsx";
+import { SegmentedPickerProps } from "./type";
+
+export const SegmentedPicker: React.FC<SegmentedPickerProps> = ({
+  className,
+  segments,
+  onSelect,
+}) => {
+  const [selectedSegment, setSelectedSegment] = useState(segments[0]);
+  const onSegmentClick = (segment: string) => {
+    setSelectedSegment(segment);
+    onSelect(segment);
+  };
+
+  return (
+    <div
+      className={clsx(
+        "flex items-center justify-between",
+        "rounded-lg bg-soda-40 px-[2px]",
+        "overflow-hidden",
+        className
+      )}
+    >
+      {segments.map((segment) => (
+        <div
+          className={clsx(
+            "flex flex-1 items-center justify-center",
+            "relative gap-1",
+            segment !== selectedSegment &&
+              ` 
```

Comment:
please don't use ` when you do not use variable inner

Reply:
change:replace `` to ""